<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>R: Analysis of molecular variance</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>

<table width="100%" summary="page for amova {ade4}"><tr><td>amova {ade4}</td><td style="text-align: right;">R Documentation</td></tr></table>

<h2>Analysis of molecular variance</h2>

<h3>Description</h3>

<p>The analysis of molecular variance tests the differences among population and/or groups of populations
in a way similar to ANOVA. It includes evolutionary distances among alleles.
</p>


<h3>Usage</h3>

<pre>
amova(samples, distances, structures)
## S3 method for class 'amova'
print(x, full = FALSE, ...)
</pre>


<h3>Arguments</h3>

<table summary="R argblock">
<tr valign="top"><td><code>samples</code></td>
<td>
<p>a data frame with haplotypes (or genotypes) as rows, populations as columns
and abundance as entries</p>
</td></tr>
<tr valign="top"><td><code>distances</code></td>
<td>
<p>an object of class <code>dist</code> computed from Euclidean distance.
If <code>distances</code> is null, equidistances are used.</p>
</td></tr>
<tr valign="top"><td><code>structures</code></td>
<td>
<p>a data frame containing, in the jth row and the kth column,
the name of the group of level k to which the jth population belongs</p>
</td></tr>
<tr valign="top"><td><code>x</code></td>
<td>
<p>an object of class <code>amova</code></p>
</td></tr>
<tr valign="top"><td><code>full</code></td>
<td>
<p>a logical value indicating whether the original data ('distances', 'samples', 'structures')
should be printed</p>
</td></tr>
<tr valign="top"><td><code>...</code></td>
<td>
<p>further arguments passed to or from other methods</p>
</td></tr>
</table>


<h3>Value</h3>

<p>Returns a list of class <code>amova</code>
</p>
<table summary="R valueblock">
<tr valign="top"><td><code>call</code></td>
<td>
<p>call</p>
</td></tr>
<tr valign="top"><td><code>results</code></td>
<td>
<p>a data frame with the degrees of freedom, the sums of squares, and the mean squares.
Rows represent levels of variability.</p>
</td></tr>
<tr valign="top"><td><code>componentsofcovariance</code></td>
<td>
<p>a data frame containing the components
of covariance and their contribution to the total covariance</p>
</td></tr>
<tr valign="top"><td><code>statphi</code></td>
<td>
<p>a data frame containing the phi-statistics</p>
</td></tr>
</table>


<h3>Author(s)</h3>

<p>Sandrine Pavoine <a href="mailto:pavoine@mnhn.fr">pavoine@mnhn.fr</a>
</p>


<h3>References</h3>

<p>Excoffier, L., Smouse, P.E. and Quattro, J.M. (1992) Analysis of molecular variance inferred
from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction
data. <em>Genetics</em>, <b>131</b>, 479&ndash;491.
</p>


<h3>See Also</h3>

<p><code><a href="randtest.amova.html">randtest.amova</a></code></p>


<h3>Examples</h3>

<pre>
data(humDNAm)
amovahum &lt;- amova(humDNAm$samples, sqrt(humDNAm$distances), humDNAm$structures)
amovahum
</pre>

<hr /><div style="text-align: center;">[Package <em>ade4</em> version 1.7-4 <a href="00Index.html">Index</a>]</div>
</body></html>
